transports/webrtc/src/tokio/certificate.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+// DEALINGS IN THE SOFTWARE.
+
+use rand::{distributions::DistString, CryptoRng, Rng};
+use webrtc::peer_connection::certificate::RTCCertificate;
+
+use crate::tokio::fingerprint::Fingerprint;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Certificate {
+    inner: RTCCertificate,
+}
+
+impl Certificate {
+    /// Generate new certificate.
+    ///
+    /// `_rng` argument is ignored for now. See https://github.com/melekes/rust-libp2p/pull/12.
```

Comment:
```suggestion
    /// `_rng` argument is ignored for now. See <https://github.com/melekes/rust-libp2p/pull/12>.
```

To make rustdoc happy.